internal/util/read_affinity.go

```diff
@@ -0,0 +1,76 @@
+// /*
```

Comment:
nit: the commenting looks very weird. can you use the same as what is done in other files?